tests/test_confloader.py

```diff
@@ -16,6 +16,9 @@
     ("--conf='test.ini'", 'test.ini'),
     ('--conf="test.ini"', 'test.ini'),
     ('--conf test.ini', 'test.ini'),
+    ('--conf     test.ini', 'test.ini'),
+    ('--conf\ttest.ini', 'test.ini'),
+    ('--conf\ntest.ini', 'test.ini'),
     ('--conf /path/to/file-test.ini', '/path/to/file-test.ini'),
     ('--conf=/path/to/file-test.ini', '/path/to/file-test.ini'),
     ("--conf='/path/to/file-test.ini'", '/path/to/file-test.ini'),
```

Comment:
Suggestion for a pathological test case:
`('--conf=conf-with-'-quote.ini', 'conf-with-'-quote.ini')`